corehq/warehouse/loaders/__init__.py

```diff
@@ -0,0 +1,57 @@
+from corehq.warehouse.loaders.app_status import (
+    ApplicationStatusFactLoader,
+    AppStatusFormStagingLoader,
+    AppStatusSynclogStagingLoader,
+)
+from corehq.warehouse.loaders.application import (
+    ApplicationDimLoader,
+    ApplicationStagingLoader,
+)
+from corehq.warehouse.loaders.domain import (
+    DomainDimLoader,
+    DomainStagingLoader,
+)
+from corehq.warehouse.loaders.domain_membership import (
+    DomainMembershipDimLoader,
+)
+from corehq.warehouse.loaders.form import FormFactLoader, FormStagingLoader
```

Comment:
nit: Can this and others be also a multiline import
```
FormFactLoader,
FormStagingLoader
```
to have the same syntax followed.

I am not sure if there is a preferred way to do this. It's in other files as well.

Reply:
I'm using `isort` to do import formatting so will leave it with whatever isort does